src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c

```diff
@@ -78,43 +70,49 @@ void EnSnowwd_Destroy(Actor* thisx, PlayState* play) {
     Collider_DestroyCylinder(play, &this->collider);
 }
 
-void func_80AF76F0(EnSnowwd* this, PlayState* play) {
+void EnSnowwd_Idle(EnSnowwd* this, PlayState* play) {
+    static Vec3f sAccel = { 0.0f, 0.0f, 0.0f };
+    static Vec3f sVelocity = { 0.0f, -4.0f, 0.0f };
+    static Color_RGBA8 sPrimColor = { 255, 255, 255, 255 };
+    static Color_RGBA8 sEnvColor = { 200, 200, 220, 0 };
     s32 pad;
     Actor* thisx = &this->actor;
-    f32 sp54;
-    Vec3f sp48;
+    f32 wobbleAmplitude;
+    Vec3f pos;
 
     if (this->collider.base.acFlags & AC_HIT) {
         this->collider.base.acFlags &= ~AC_HIT;
         Actor_PlaySfx(thisx, NA_SE_IT_REFLECTION_WOOD);
     }
     if (thisx->home.rot.y != 0) {
-        this->unk190 = 0x15;
+        this->timer = 21;
         thisx->home.rot.y = 0;
-        if (thisx->home.rot.z == 0) {
-            if (SNOWWD_GET_PARAM_F80(&this->actor) < 0x10) {
-                sp48 = thisx->world.pos;
-                sp48.y += 200.0f;
-                Item_DropCollectibleRandom(play, NULL, &sp48, SNOWWD_GET_PARAM_F80(&this->actor) * 0x10);
+        if (SNOWWD_DROPPED_COLLECTIBLE(&this->actor) == 0) {
+            if (SNOWWD_GET_DROP_TABLE(&this->actor) < DROP_TABLE_SIZE) {
```

Comment:
```suggestion
            if (SNOWWD_GET_DROP_TABLE(&this->actor) < 16) {
```
🤔 Hmm thinking about this again, `DROP_TABLE_SIZE` actually doesn't make sense here, it should refer to the `DROP_TABLE_NUMBER` instead. Probably best to just set it back to 16 for now. (Note the `DROP_TABLE_SIZE` below does still make sense and should be left).